### PR TITLE
Added CreateClientFromEnv and imports to python generation

### PR DESCRIPTION
**Pull request by RachelTucker**

### CODE REVIEW

File: ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+     Required: DS3_ACCESS_KEY, DS3_SECRET_KEY, DS3_ENDPOINT
+
+     Optional: http_proxy
+  """
+  access_key = os.environ.get('DS3_ACCESS_KEY')
+  secret_key = os.environ.get('DS3_SECRET_KEY')
+  endpoint = os.environ.get('DS3_ENDPOINT')
+  proxy = os.environ.get('http_proxy')
+
+  if None in (access_key, secret_key, endpoint):
```
Comment (rpmoore) [inline ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl]: I didn't realize you could do that.  That's really cool.

File: ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl [hunk 2]
```diff
@@ -48,8 +69,7 @@ class AbstractResponse(object):
   def __check_status_codes__(self, expected_codes):
     if self.response.status not in expected_codes:
       err = "Return Code: Expected %s - Received %s" % (expected_codes, self.response.status)
-      ds3error = XmlSerializer().to_ds3error(self.response.read())
-      raise RequestFailed(err, ds3error)
+      raise RequestFailed(err, self.response)
```
Comment (rpmoore) [inline ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl]: Does RequestFailed know how to parse the error response?

Comment (RachelTucker) [inline ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl]: The current code for RequestFailed is:

class RequestFailed(Exception):
  def __init__(self, summary, response):
    self.summary = summary
    self.code = response.status
    self.reason = response.reason

  def __str__(self):
    return '{0} \n Code={1} \n {2}'.format(self.summary, self.code, self.reason)

Comment (rpmoore) [inline ds3-autogen-python/src/main/resources/tmpls/python/commands/all_commands.ftl]: So we're not really parsing it at all.  We should look enhancing that in the future.

**Pull request merged**